src/azul/service/responseobjects/elastic_request_builder.py

```diff
@@ -420,7 +433,7 @@ def transform_summary(
             ('contents.specimens.document_id', 'specimenCount'),
             ('contents.files.uuid', 'fileCount'),
             ('contents.donors.document_id', 'donorCount'),
-            ('contents.projects.laboratory', 'labCount'),
+            ('contents.projects.laboratory', 'labCount'),   # FIXME Is this now counting '__null__' as a value?
```

Comment:
Probably, but I am just going to say that that's ok.

Reply:
Created new ticket #1188 to address this issue